src/helpers/UppyFileTypeCheckerPlugin.js

```diff
@@ -12,6 +12,9 @@ export const getAllowedFileTypes = (task) => {
           mimeTypes: ['audio/*', 'video/*'],
         };
       case image_classification:
+      case visualQuestionAnswering:
+      case textGuidedImagetoImage:
+      case documentQuestionAnswering:
```

Comment:
Should `documentQuestionAnswering` (and these others) actually accept all of these file types?